test/e2e/common/init_container.go

```diff
@@ -149,12 +148,6 @@ var _ = framework.KubeDescribe("InitContainer [NodeConformance]", func() {
 					{
 						Name:  "run1",
 						Image: imageutils.GetPauseImageName(),
-						Resources: v1.ResourceRequirements{
-							Limits: v1.ResourceList{
-								v1.ResourceCPU:    *resource.NewMilliQuantity(100, resource.DecimalSI),
-								v1.ResourceMemory: *resource.NewQuantity(50*1024*1024, resource.DecimalSI),
```

Comment:
Why don't we remove memory limit line only without moving CPI limit line?
Two previous PRs which updated here are only for memory limitation quantity and OOM is memory issue.

Reply:
Just removed memory only.